back-end/app.js

```diff
@@ -15,7 +15,7 @@ const server = http.createServer(app);
 // Middlewares
 const authentication = require('./middlewares/authentication');
 const cors = require('./middlewares/cors');
-const io = new Server(server, { cors: { origin: 'https://incredible-torte-ac738e.netlify.app' } });
+const io = new Server(server, { cors: { origin: process.env.SOCKET_FRONT_URL } });
```

Comment:
Maybe, better try this app.settings.env. In env we should have some secure values.